source/includes/markdown/_13_1-scim.html.md

```diff
@@ -3,25 +3,52 @@
 
 # SCIM
 
-## SCIM endpoints
 
 Asana supports SCIM operations at `https://app.asana.com/api/1.0/scim`. Okta provides greats docs for 
 [understanding SCIM](https://developer.okta.com/docs/concepts/scim/).
 
 Note that only Service Accounts in Enterprise Domains can access SCIM endpoints.
 
+<br />
+
+## Service Provider Configuration Endpoints
+
 |API call|Asana Behavior|
 |--------|--------------|
-|GET /Users|Return full list of users in the domain.  Does not return Asana guest users|
-|GET /Users/:id|Return specific user in the domain.  Does not return Asana guest users.|
-|POST /Users|Creates new user if the user does not exist|
-|PUT /Users/:id|Update / remove attributes for a user. Deprovision user (zombify) in Asana upon active=false|
-|PATCH /Users/:id|Deprovision user (zombify) in Asana upon active=false|
-|DELETE /Users/:id|Deprovision user (zombify) in Asana|
 |GET /ServiceProviderConfig|Read-only meta information|
 |GET /ResourceTypes|Read-only meta information|
 |GET /Schemas|Read-only meta information|
 
+<br />
+
+## SCIM Endpoints
+
+### User Resource
+
+|API call|Asana Behavior|
+|--------|--------------|
+|GET /Users|Return full list of users in the domain.  Does not return Asana guest users.|
+|GET /Users/:id|Return specific user in the domain.  Does not return Asana guest users.|
+|POST /Users|Create a new user if the user does not exist.|
+|PUT /Users/:id|Update / remove attributes for a user. Deprovision user (zombify) in Asana upon active=false.|
+|PATCH /Users/:id|Deprovision user (zombify) in Asana upon active=false.|
+|DELETE /Users/:id|Deprovision user (zombify) in Asana.|
+
+<br />
+
+### Group Resource
+Note that SCIM groups are equivalent to Asana Teams.
+
+|API call|Asana Behavior|
+|--------|--------------|
+|GET /Groups|Return full list of teams in the domain, including private teams.|
+|GET /Groups/:id|Return specific team in the domain. |
+|POST /Groups|Create a new team.|
+|PUT /Groups/:id|Update / remove attributes for a team.|
+|PATCH /Groups/:id|Update the team's attributes.|
+|DELETE /Groups/:id|Remove a team in Asana.|
```

Comment:
This endpoint is not implemented. We should either call that out or remove it from this list.

Reply:
Will remove it for now 👍